AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -75,7 +75,7 @@ public class AnalyzeCommand
         private DateTime _lastUpdated;
 
         //save all unique dependencies even if Dependency tag pattern is not-unique
-        private Regex tagPatternRegex = new Regex("Dependency.SourceInclude", RegexOptions.IgnoreCase);
+        private Regex tagDepPatternRegex = new Regex("Dependency.SourceInclude", RegexOptions.IgnoreCase);
```

Comment:
If this runs a lot you should pass `RegexOptions.Compiled`

Reply:
You don't need to use a Regex here -- just do a string contains. If there's another reason for using the regex, just escape the period in the middle so it isn't interpreted.